README.md

```diff
@@ -117,13 +117,30 @@ The Python integration tests make use of the VMM in varied configurations that
 aren’t overly complex and illustrate realistic use cases (e.g. one test runs a
 VMM with a virtio block device, one test will run a VMM with PCI, etc.).
 
-To be able to successfully run all the tests in this repo, there are a couple of
-resources that need to be found in `/tmp`. To generate the resources at the
-expected locations, you can run the following command:
+### Dependencies for Testing
 
-```bash
-sudo ./resources/make_test_resources.sh
-```
+To be able to successfully run all the tests in this repo, pre-created
+resources are stored in S3. The resources are downloaded locally inside the
+`resources` directory. This is handled transparently by the test cases.
+Note: The resources once downloaded are cached, so they are not downloaded on
+every run.
+
+Running test requires `Python3` and the following dependencies to be installed.
+- `boto3` for downloading resources from S3 mentioned above.
+- `pytest` for running the VMM reference tests.
+
+Please use your distribution specific commands to download this.
+
+For example on Ubuntu based system you can run the following commands to install
+required dependencies -
+`sudo apt-get install python3-boto3 python3-pytest`
+
+or
+
+if you prefer using `pip` to install the dependencies -
+`pip install boto3 pytest`
```

Comment:
This is not the recommended way of running the tests. We recommend running the tests with the container which already has all these tools installed. The reason is that tests might fail even if you have all the tools installed if the versions are for example different. Can you please add a sub-section with "Running without the rust-vmm container" and explicitly call out that this is not the recommend way?